sceptre/plan/actions.py

```diff
@@ -1141,3 +1141,10 @@ def _describe_stack_resource_drifts(self) -> dict:
                 "StackName": self.stack.external_name
             }
         )
+
+    @add_stack_hooks
+    def list_config(self, config_reader):
+        """
+        Returns the Template for the Stack
+        """
+        return config_reader.read(self.stack.name + ".yaml")
```

Comment:
@jfalkenstein I am not following because, as far as I know, `self.stack.name + ".yaml"` is a file name, not a path:

```
ipdb> self.stack.name + ".yaml"
'aurora.yaml'
```